src/fit-bounds.js

```diff
@@ -3,10 +3,13 @@
 import Transform from 'mapbox-gl/js/geo/transform';
 import {LngLatBounds, Point} from 'mapbox-gl';
 
-export default function fitBounds(width, height, _bounds, options) {
-  const bounds = new LngLatBounds([_bounds[0].reverse(), _bounds[1].reverse()]);
-  options = options || {};
-  const padding = typeof options.padding === 'undefined' ? 0 : options.padding;
+export default function fitBounds(width, height, _bounds, {
+  padding = 0
```

Comment:
not a big deal but I would have put this in a seperate PR since it changes the functionality of `fitBounds`.